_opensearch/query-dsl/geo-and-shape.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Geographic and shape queries
+parent: Query DSL
+nav_order: 70
+---
+
+# Geographic and shape queries
+
+You can search multiple dimensions with the geographic and shape query types. You can create queries to search two-dimensional geometries that map out cartesian data such as `point` fields that support x and y pairs and `shape` fields that support points, lines, circles and polygon shapes.
+
+## Geographic queries
+
+You can use the following geographic queries to search documents that include `geo_point` field data:
+
+- **Geo-bounding box** `geo_bounding_box` – Returns documents with `geo_point` field values that are within a bounding box.
+- **Geo distance** `geo_distance` – Returns documents that contain the geographic points within a specified distance from a central point.
+- **Geo polygon** `geo_polygon` – Returns documents that contain the specified geographic points within the specified polygon.
+- **xy point queries** – Returns documents that contain two-dimensional coordinates in `xy_point` or `xy_shape` fields.  To learn more, see [xy queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/xy/).
+
+## Pre-indexed shape query
```

Comment:
This is applicable for xy point queries and (I think) geoshape queries only. We can see how similar the parameters are and decide whether to keep this section here.